app/views/teams/_conference_info.html.slim

```diff
@@ -15,10 +17,26 @@ h3 Conferences
 
   - if @team.conference_attendances.present?
     h4 Conference Offers
+    .conference-info
+      | Free tickets offered by RGSoC. Please accept of reject the offer(s).
+
     ul.conferences
       - @team.conference_attendances.each do |offer|
         li
           = links_to_conferences([offer.conference]).first
+          - if offer.attendance.present?
+          = show_attendance offer.attendance
+
         - if offer.orga_comment.present?
-        |Orga comment:
-        =offer.orga_comment
\ No newline at end of file
+        .orgaComment
+          |Orga Comment:
+          p = offer.orga_comment
+
+        - if can? :accept_or_reject_conference_offer, @team
+          ul.attendance.list-inline
+            -if offer.attendance != true
+              li
+                = link_to "Yes, I can attend", conference_attendance_path(offer, attendance: :true, team_id: @team.id), method: :put, class: 'btn btn-default'
```

Comment:
I would probably try to nest the params here or use an inline form. If I'm not mistaken this currently generates a path with a bunch of get params and then submits them via put. I think it would be more elegant to have these be submitted as post params.